<html>
<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>
<body>
    <tt>
        BMC Bioinformatics. 2010; 11: 410. <br/>
        Published online 2010 August 3. doi:  10.1186/1471-2105-11-410 <br/>
        PMCID: PMC2923139
    </tt>    
        <h1>A method for automatically extracting infectious disease-related primers and probes from the literature</h1>
        <h5>
            Miguel Garcia-Remesal<sup>#1,2</sup> Alejandro Cuevas<sup>#2</sup> 
            Victoria Lopez-Alonso<sup>#2</sup> Guillermo Lopez-Campos<sup>#2</sup> 
            Guillermo de la Calle<sup>#2</sup> Diana de la Iglesia<sup>#2</sup> 
            David Perez-Rey<sup>#1,2</sup> Jose Crespo<sup>#2</sup> 
            Fernando Martin-Sanchez<sup>#2</sup> and Victor Maojo<sup>#1,2</sup>.
        </h5>
        <h6>
            #1 corresponding author<br/>
            #2 coauthor
        </h6>  
        <h2><font color="#22CC77">Abstract</font></h2> 
        <hr color="#22CC77">
        <h3><i>Background</i></h3>
        <p>
            Primer and probe sequences are the main components of nucleic acid-based 
            detection systems. Biologists use primers and probes for different tasks, 
            some related to the diagnosis and prescription of infectious diseases. 
            The biological literature is the main information source for empirically 
            validated primer and probe sequences. Therefore, it is becoming increasingly 
            important for researchers to navigate this important information. In this paper, 
            we present a <i>four-phase method</i>  for extracting and annotating primer/probe 
            sequences from the literature. These phases are: <br/>(1) convert each document 
            into a tree of paper sections, <br/>(2) detect the candidate sequences using 
            a set of finite state machine-based recognizers, <br/>(3) refine problem sequences 
            using a rule-based expert system, and <br/> (4) annotate the extracted sequences 
            with their related organism/gene information.
        </p>
        <h3><i>Results</i></h3>
        <p>
            We tested our approach using a test set composed of 
            <font color="#22CC77"><i>297 manuscripts</i></font>. The 
            extracted sequences and their organism/gene annotations were manually 
            evaluated by a panel of molecular biologists. The results of the evaluation 
            show that our approach is suitable for automatically extracting <acronym title="Acido Desoxirribonucleico">DNA</acronym> sequences, 
            achieving precision/recall rates of 97.98% and 95.77%, respectively. In 
            addition, 76.66% of the detected sequences were correctly annotated with 
            their organism name. The system also provided correct gene-related information 
            for 46.18% of the sequences assigned a correct organism name.
        </p>
        <h3><i>Conclusions</i></h3>
        <p>
            We believe that the proposed method can facilitate routine tasks for biomedical 
            researchers using molecular methods to diagnose and prescribe different 
            infectious diseases. In addition, the proposed method can be expanded to detect 
            and extract other biological sequences from the literature. The extracted 
            information can also be used to readily update available primer/probe databases 
            or to create new databases from scratch.
        </p>        
</body>
</html>
